src/pages/chainlink-nodes/contracts/operator.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: nodeOperator
+date: Last Modified
+title: "Operator"
+---
+
```

Comment:
I think somewhere in this page we should make a reference to the added security guarantee for having operator owned by hardwallet managing EOAs. This is useful for NOPs to understand how Operator is useful outside of  it being a drop-in replacement for oracle contract.